tests/vc-generator/validVc.js

```diff
@@ -3,12 +3,11 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 export const validVc = {
-  '@context': ['https://www.w3.org/2018/credentials/v1'],
+  '@context': ['https://www.w3.org/ns/credentials/v2'],
   id: 'urn:uuid:86294362-4254-4f36-854f-3952fe42555d',
   type: ['VerifiableCredential'],
-  issuer: 'did:key:z6MksNZwi2r6Qxjt3MYLrrZ44gs2fauzgv1dk4E372bNVjtc',
-  issuanceDate: '2020-03-16T22:37:26.544Z',
   credentialSubject: {
-    id: 'did:key:z6MktKwz7Ge1Yxzr4JHavN33wiwa8y81QdcMRLXQsrH9T53b'
+    id: 'did:key:z6MktKwz7Ge1Yxzr4JHavN33wiwa8y81QdcMRLXQsrH9T53b',
+    name: 'ExampleClaim'
```

Comment:
This one still worries me a bit as it's slightly above the minimum requirements--though likely a very common field even in a minimal VC. I'm open to not caring...but right now it does give me pause...

Reply:
From my point of vue, name/description are globally accessible properties in vcdm 2.0. I'm personally ok with including this.